src/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Person from './Person';
+
+const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
+
+const App = () => (
+  <div className="App">
+    <Person {...misha} />
```

Comment:
Don't spread props like this. Write them individually or pass the whole objedct.